static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -159,18 +170,40 @@ class IssueRuleEditor extends AsyncView<Props, State> {
     return createFromDuplicate && location?.query.duplicateRuleId;
   }
 
+  componentWillMount() {
+    this.fetchPreview();
+    this.fetchMemberList();
+  }
+
   componentWillUnmount() {
     window.clearTimeout(this.pollingTimeout);
   }
 
   componentDidUpdate(_prevProps: Props, prevState: State) {
+    if (prevState.previewCursor !== this.state.previewCursor) {
+      this.fetchPreview();
+    } else if (this.isRuleStateChange(prevState)) {
```

Comment:
fetchPreview will need to be debounced, we shouldn't make a new request on every keystroke